flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/SequenceFileSerializerFactory.java

```diff
@@ -36,16 +38,14 @@
   @SuppressWarnings("unchecked")
   static SequenceFileSerializer getSerializer(String formatType,
                                               Context context) {
-
-    Preconditions.checkNotNull(formatType,
-        "serialize type must not be null");
+    Preconditions.checkNotNull(formatType, "serialize type must not be null");
 
     // try to find builder class in enum of known formatters
     SequenceFileSerializerType type;
     try {
       type = SequenceFileSerializerType.valueOf(formatType);
     } catch (IllegalArgumentException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1166)